reporter/src/main/resources/templates/asciidoc/vulnerability_report.ftl

```diff
@@ -23,7 +23,11 @@
 :sectnums:
 :toc:
 
-= Vulnerability Report
+[#assign firstProjectId = projects?first.id]
+[#assign firstProjectUrl = ortResult.getProject(firstProjectId).vcsProcessed.url]
```

Comment:
Commit message: "Use the **first** project's name..." to emphasize that the choice is a bit arbitrary.
Off-topic, but I think we need a way to set a name for an ORT run, would be useful here and in many other places. Could default to the Git repository or directory name, for example.

Reply:
> Commit message: "Use the **first** project's name..." to emphasize that the choice is a bit arbitrary.

Done.

> Off-topic, but I think we need a way to set a name for an ORT run, would be useful here and in many other places. Could default to the Git repository or directory name, for example.

Agreed. This relates to our old discussion about what a "project" should be: A technical / build system project like ORT uses the term now, or a "product owner project", which could be compromised of multiple (e.g. frontend and backend) technical projects.